testing/tests/physics.lua

```diff
@@ -307,10 +307,10 @@ love.test.physics.Joint = function(test)
   test:assertFalse(joint:isDestroyed(), 'check not destroyed')
 
   -- check reaction props (sometimes nil on linux runners)
-  if joint:getReactionForce(1) ~= nil then
+  if joint:getReactionForce(1) ~= nil and joint:getReactionForce(1) ~= 0/0 then
```

Comment:
Do we know why this is failing sometimes? I think if there's a random failure on CI machines like this I'd rather either investigate it or have this part of the test very obviously temporarily turned off until it can be investigated.

Reply:
Answer: I can't find any situation where getReactionForce is able to return nil in the code, but the numbers it does return (as well as possible other Joint getters) are not valid until the World has been updated at least once after the Joint is created, I think.

Reply:
yeah ideally I'd like to leave if it's a failure but it's been so random, I'd say maybe 1/20 of the workflow actions I've run I've seen it popup

I'll try doing a world update before those joint calls, and remove the ~= nil ~= 0/0 checks to let it fail and we can see if having the world update solves it happening - if not can decide what you wanna do